README.md

```diff
@@ -2,8 +2,20 @@
 
 A new implementation of the webcompoents.org site
 
-This repo will contain several packages:
+This repo contains several packages:
 
-* A data-only backend that indexes npm packages and provides a GraphQL API into the database
-* A frontend server that serves the user-facing webcompoents.org site
-* An HTML client served by the frontend server
+- `wc-registry`: A data-only backend that indexes npm packages and provides a GraphQL API into the database
```

Comment:
I'm going to change all of these packages to be under the `@webcomponents` scope.